pkg/apis/crd/v1alpha1/types.go

```diff
@@ -23,7 +23,7 @@ import (
 type TraceflowPhase string
 
 const (
-	// Pending is not used anymore
+	// Deprecated: Pending is not used anymore
```

Comment:
I don't think it's reasonable to refine comment for IDE behavior since different IDEs may have different requirements.  @antoninbas @tnqn  any comments?

Reply:
If it's necessary, I may include the change in PR https://github.com/antrea-io/antrea/pull/5108.

Reply:
This is a very common convention. Please see https://zchee.github.io/golang-wiki/Deprecated.

"[Some tools will warn on use of deprecated identifiers](https://staticcheck.io/docs/checks#SA1019) and their docs [are hidden on pkg.go.dev](https://go.dev/issue/40850)."

Reply:
This looks fine to me.
@luolanzone we can merge this PR as is, and you can remove `Pending` altogether for `v1beta1/types.go` in your PR